test/manual/amp-analytics/performanceTestPages/visible-shorter.html

```diff
@@ -0,0 +1,470 @@
+<!DOCTYPE html>
```

Comment:
Could you please explain the difference between the two examples. Thanks

Reply:
Added comments to the top of the files (and also renamed).
- `visible-basic` imitates a simple article with images and text 
- `visible-multiple` imitates a longer page with multiple blocks of text, images, and ads

Reply:
sounds good. thanks